modules/weko-admin/weko_admin/utils.py

```diff
@@ -859,30 +878,39 @@ def send_mail(cls, recipient, body, subject):
         return MailSettingView.send_statistic_mail(rf)
 
     @classmethod
-    def build_statistic_mail_subject(cls, title, send_date):
+    def build_statistic_mail_subject(cls, title, send_date,
+                                     system_default_language):
         """Build mail subject.
 
         Arguments:
             title {string} -- The site name
             send_date {string} -- statistic time
+            system_default_language {string} -- default language.
 
         Returns:
             string -- The mail subject
 
         """
-        result = '[' + title + ']' + send_date
-        if get_system_default_language() == 'ja':
+        if title:
+            result = '[' + title + ']' + send_date
```

Comment:
I think, we don't need to check title at here, because we had "title" default is equal to "No Site Name".